DQM/SiPixelPhase1Track/plugins/SiPixelPhase1TrackClusters.cc

```diff
@@ -86,6 +94,8 @@ namespace {
     edm::ESGetToken<TrackerTopology, TrackerTopologyRcd> trackerTopoToken_;
     edm::ESGetToken<TrackerGeometry, TrackerDigiGeometryRecord> trackerGeomToken_;
     edm::ESGetToken<ClusterShapeHitFilter, CkfComponentsRecord> clusterShapeHitFilterToken_;
+
+    edm::ESGetToken<SiPixelTemplateDBObject, SiPixel2DTemplateDBObjectESProducerRcd> templateDBobjectToken_;
```

Comment:
```suggestion
    edm::ESGetToken<SiPixelTemplateDBObject, SiPixelTemplateDBObjectESProducerRcd> templateDBobjectToken_;
```
@arossi83 in addition this should not be the 2D record, but the regular 1D template record